### ISSUE TITLE
Insufficient color contrast (especially for colorblindness)

### ISSUE THREAD

**Issue by schalkventer** [ShailenNaidoo/Netlify#6]
![image](https://user-images.githubusercontent.com/14258328/65836091-d9761880-e2ee-11e9-84d7-4dc5844b9ec0.png)

**Comment by ShailenNaidoo** [comment]
@schalkventer I think you would be the best person to come up with a good color palette as I am color deficient so I am not the best with colors :sweat_smile:

**Comment by martinwheeler** [comment]
I'm happy to have a look at this if no one else is 😊🤓

**Comment by ShailenNaidoo** [comment]
Sure go ahead! :)

**Comment by ShailenNaidoo** [comment]
> I'm happy to have a look at this if no one else is

@martinwheeler Hey man! Are you still keen on doing this issue? :smile:

**Comment by martinwheeler** [comment]
> > I'm happy to have a look at this if no one else is
> 
> @martinwheeler Hey man! Are you still keen on doing this issue? 😄

Sorry @ShailenNaidoo, I let this one slip rather badly! I'll get together a PR this week 😄

**Comment by ShailenNaidoo** [comment]
@martinwheeler Thanks man! Would appreciate your contribution. I am color deficient so I am the wrong person to pick colors. :sweat_smile:

**Comment by ShailenNaidoo** [comment]
@martinwheeler v0.6.4 was published! Added your name and Github to the changelog :smile:
